src/app/Login/page.tsx

```diff
@@ -0,0 +1,91 @@
+"use client"
+import { useState } from "react"
+import { Container, Typography, TextField, Button, Snackbar } from "@mui/material"
+import { useNavigate } from "react-router-dom"
+import './Login.css'
+
+function Login() {
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [error, setError] = useState<string | null>(null)
+    const [openSnackbar, setOpenSnackbar] = useState<boolean>(false)
+    //const navigate = useNavigate()
+
+    const handleLogin = () => {
+        fetch('../api/mock/user/login/route', {
```

Comment:
```suggestion
        fetch('/api/mock/user/login', {
```
api 요청은 실제 서버에 보내듯이 이렇게 보내면 됩니다! 절대경로로 요청 보내시면 되어요

+) 저희는 react-query를 사용하니 useMutation 사용해봐도 좋을 것 같네요